.github/workflows/pre_release.yml

```diff
@@ -35,6 +35,10 @@ jobs:
       - name: Bump Versions
         run: deno run -A ./console/bumper_ci_service.ts --version=${{ github.ref }} # vrsion passed so we know what the new release version will be
 
+      - name: Send Event to Website to Update Versions
+        run: |
+          curl -XPOST -u "${{ secrets.CI_USER_NAME }}:${{ secrets.CI_USER_PAT }}" -H "Accept: application/vnd.github.everest-preview+json" -H "Content-Type: application/json" https://api.github.com/repos/drashland/website/dispatches --data '{"event_type": "update_docs", "client_payload": { "module_to_update": "drash", "version": "${{ github.ref }}" }}'
```

Comment:
can we break this apart into separate lines 😄

Reply:
nvm. idk if breaking it apart into separate lines would work the first time and i want this to work the first time. resolving.